### ISSUE TITLE
[a11y] My collection page - missing labels

### ISSUE THREAD

**Issue by jxjj** [UMN-LATIS/z#227]
https://z.umn.edu/shortener/groups

"Collection Name" and "Description" (add new collection) fields not labeled.

Audit issue 29
